### ISSUE TITLE
Again "Spawn ctest ENOENT"

### ISSUE THREAD

**Issue by Gibson85** [vector-of-bool/vscode-cmake-tools#465]
This is related to issue #463 and #428.

I know, my previous issue/complain wasn't very nice and being angry doesn't help much! But you have to know I had reported several bugs in the past (other accounts) and even after almost a year nothing was done to solve them from you although these bugs had been very obvious and easy to reproduce. And only because an update to the newest cmake version made this plugin working again I changed back from Atom to VSCode in the beginning of this year.

@Randshot
> @Gibson85 Have you looked up what spawn * ENOENT means?
It means that it couldn't find the command/executable in question, so this doesn't really have anything to do with the extension itself.
You could argue that unhelpful error messages are a kind of bug, but that is another issue.

Yes I googled for it but found no solution and in contrast to you I am pretty sure it is a bug of this plugin, because this was a reported bug #428 from your own collaborateur in the past and vector-of-bool proved this being a bug. So please before saying it's not a bug have at least a damn SINGLE TEST that proves your statements. THANKS. Again PLEASE do better testing in future. These bugs are not acceptable any more. And if versions 0.11.X worked for months without any problems and the update to version 1.0.0 broke everything (once again) how else should I call it if not a bug?

@vector-of-bool 
Yes, I found the bug report #428 over google before and I saw you claimed you would have fixed it with version 1.0.1. But it's obviously still not fixed.

As I wrote before I have no hope any more this will be fixed or at least fixed soon, because Linux seems not to have any focus from you guys. I know getting positive feedback is always better and I wish as a user from the very first beginnings of this plugin I could give so. But after all this time and still so many fatal bugs with every bigger release I am more than annoyed from this plugin. All these obvious and very easy to reproduce errors show you never had a single test on a linux platform. I verified the other bugs I reported before on at least 2 different distributions (Debian & Fedora). A single start of vscode or single try to build would have shown these bugs.

Nevertheless thanks for this plugin and I really hope I can use it in future again.

To the bug:
At least if I delete the Debug/Release folders compiling works again. But compiling these from new takes me 2 hours per day for nothing...

**Comment by vector-of-bool** [comment]
I took a look into the relationship between your problem and #428.

#428 _was_ fixed, but it's fixing actually _caused_ your issue. The problem is that moving `cmakePath` doesn't automatically fix `ctestPath`. The fix was to default `ctestPath` to look in the same directory as the found `cmakePath`.

Based on my intuition, I guess that you have a CMake executable in a directory without a corresponding CTest executable.

This situation is very unique: most people install all of CMake, CTest, and CPack in the same directory.

I wrote a test that reproduces your issue and I added a fix. It will be available in the next release.

What other issues have you been waiting on for so long? I've been focusing on stabilizing recently and I'd like to get my issue count down.
